docs/Theming.md

```diff
@@ -409,6 +409,22 @@ const MyAppBar = props => <AppBar {...props} userMenu={<MyUserMenu />} />;
 const MyLayout = props => <Layout {...props} appBar={MyAppBar} />;
 ```
 
+`AppBar` can accept children so you are able to add more custom components in it.
```

Comment:
This should probably go inside the `Using a Custom AppBar` section.

Reply:
👍

Reply:
will change it asap

Reply:
done